client/my-sites/plan-features/actions.jsx

```diff
@@ -69,7 +70,11 @@ const PlanFeaturesActions = ( {
 		if ( isLandingPage ) {
 			buttonText = translate( 'Select', { context: 'button' } );
 		}
-		if ( isInSignup ) {
+		if ( isLaunchPage ) {
+			if ( freePlan ) {
+				buttonText = translate( 'Keep this plan', { context: 'button' } );
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.